src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -613,14 +613,20 @@ private Color InternalThumbColor
         {
             get
             {
-                return thumbImage?.Color;
+                return thumbColor;
             }
             set
             {
                 if (null != thumbImage)
                 {
-                    thumbImage.BackgroundColor = value;
                     thumbColor = value;
+
+                    PropertyMap map = new PropertyMap();
+                    map.Insert((int)Visual.Property.Type, new PropertyValue((int)Visual.Type.Color));
+                    map.Insert((int)ColorVisualProperty.MixColor, new PropertyValue(thumbColor));
+                    map.Insert((int)Visual.Property.CornerRadius, new PropertyValue(0.5f));
+                    map.Insert((int)Visual.Property.CornerRadiusPolicy, new PropertyValue((int)VisualTransformPolicyType.Relative));
+                    thumbImage.Image = map;
```

Comment:
Hello, @Samsung/nui guys!
I'd like to get your opinions.
This change is currently to set `Image` property of `ImageView`. However, it can be `Color` property of `View` instead.
```
                if (null != thumbImage)
                {
                    thumbColor = value;
                    thumbImage.Color = value;
                }
```

But, when I tested with `Color` property, the image quality is a little low at first before moving the thumb.
So, I chose `Image` property like this.
What do you think?
This code is to create PropertyMap instance locally and change color visual.

Reply:
`thumbImage.Color` is the color of Actor.. so If we setup like this, the result color becom "mixed" as "original image" x "Color".

I think we need to think about the fixed-way.

1. To make similar grammer as `View.BackgroundImage` vs `View.BackgroundColor`
2. Make default thumbImage color as white, and make default thumbColor as blue, so just keep to use this property as "mixed" role.

Reply:
As we discussed, I use `Image` property as it is now. (it means `ImageVisual` for thumb color)
Just updated to remove warning messages!